doc/MANGO.tex

```diff
@@ -268,42 +268,6 @@ \subsection{Properties}
         \label{fig:property}
       \end{figure}
 
-\subsubsection{Supported Properties}
```

Comment:
why do you remove my table? It must saty in place before this subsection is fleshed up. Can you restore it?

Reply:
j ai un eu pb de synchro . git clone n a pas été cherché les dernières motifs . 
je clos la PR .